src/sagas/repository.js

```diff
@@ -430,6 +430,23 @@ export function* countResults({ payload }) {
   }
 }
 
+/**
+ * feature flag stuff
+ */
+
+export function* getFeatures() {
+  try {
+    const url = 'https://sam.dsde-dev.broadinstitute.org/api/groups/v1';
```

Comment:
worth double checking, but using dev sam makes sense to me for now

Reply:
yep, checked with @fboulnois and it seems like we're just using dev for now